classes/Gems/MenuNew/RouteHelper.php

```diff
@@ -148,27 +153,45 @@ public function getRelatedRoute(string $current, string $relative): ?string
     }
 
     /**
-     * @return string[]
+     * @return mixed[]
      */
     public function getAllRoutePrivileges(): array
     {
         return self::getAllRoutePrivilegesFromConfig($this->routes);
     }
 
     /**
-     * @return string[]
+     * @return mixed[]
```

Comment:
In feite toch nog steeds `string[]`?

Reply:
klopt, ik had daar eerst de Privilege ingezet, maar later gekozen om de label daar al neer te zetten!